application/common/helpers/MySqlDateTime.php

```diff
@@ -3,15 +3,21 @@
 
 class MySqlDateTime
 {
-    const MYSQL_FORMAT = 'Y-m-d H:i:s';
+    const MYSQL_DATETIME_FORMAT = 'Y-m-d H:i:s';
+    const MYSQL_DATE_FORMAT     = 'Y-m-d';
```

Comment:
@wcjr and I talked about that in the DB we would store only date, but we would return from the api the date plus 23:59:59 UTC so from a display standpoint we could be clear about exactly when it expires. I think Billy forgot about that and I'm not sure it's worth changing again since it is just hard coded information anyway. From an end user view we can add that it expires just before midnight UTC.